app/locales/ht.json

```diff
@@ -30,13 +28,14 @@
     "authorities_removal_alert_desc": "Eske ou si ke ou vle retire sous done otorite sa?",
     "authorities_removal_alert_proceed": "Kontinye",
     "authorities_removal_alert_title": "Retire otorite",
-    "authorities_title": "Sous ou ka fè konfyans",
     "choose_provider_subtitle": "Pou kapab toujou rete enfòme a ekspozisyon yo, ou bezwen enskri nan yon otorite.",
     "choose_provider_title": "Chwazi otorite a",
     "commitment": "Angajman",
     "commitment_para": "Chemen san danje se yon mouvman mondyal pou devlope zouti gratis, louvri sous, konfidansyalite pa konsepsyon pou rezidan yo, ofisyèl sante piblik yo, ak pi gwo kominote pou aplodi koub COVID-19, redwi laperèz, ak anpeche yon repons siveyans-eta a pandemi an.\n\nPou plis enfòmasyon, tanpri vizite ",
+    "covid_safe_paths": "COVID Safe Paths",
     "default_news_site_name": "Nouvèl sou Chase Kowona (\"Safe Paths\")",
     "event_history_subtitle": "Konprann ekspozisyon pèsonèl ou daprè enfòmasyon Otorite a pataje.",
+    "event_history_settings_title": "",
```

Comment:
is this intentional, and does it work, I recal before the react hook. empty strings broke things.

Reply:
<img width="379" alt="Screen Shot 2020-05-14 at 2 14 26 PM" src="https://user-images.githubusercontent.com/20157849/81975822-3956d080-95ed-11ea-9a68-d12640ab7850.png">

This seems to fall back to the English setting successfully.

Reply:
Did a smoke test. I was worried the fallback would error. but it worked fine.